step.yml

```diff
@@ -139,87 +133,49 @@ inputs:
         to iTunes Connect and only the review submission failed.
       value_options: ["yes", "no"]
       is_required: true
-      is_expand: false
   - skip_metadata: "yes"
     opts:
       title: "Skip Metadata?"
-      summary: ""
-      description: |
-        Flag to skip uploading the appstore metadata.
+      description: Don't upload the metadata. This will still upload screenshots.
       value_options: ["yes", "no"]
       is_required: true
-      is_expand: false
   - skip_screenshots: "yes"
     opts:
       title: "Skip Screenshots?"
-      summary: ""
-      description: |
-        Flag to skip uploading changes to appstore screenshots.
+      description: Don't upload the screenshots.
       value_options: ["yes", "no"]
       is_required: true
-      is_expand: false
-  - team_id: ""
-    opts:
-      title: "iTunes Connect: Team ID"
-      summary: ""
-      description: |-
-        The App's *Team ID* on iTunes Connect.
-
-        This field is required when using iTunes Connect account
-        linked to multiple publish team.
-
-        Use this input or `team_name` to specify the team.
-  - team_name: ""
+  - gemfile_path: ./Gemfile
```

Comment:
I'd leave this empty by default, if user specified it should be used, otherwise latest